### PR TITLE
add Extents as input to map + test

### PR DESCRIPTION
**Pull request by alex-s-gardner**

### CODE REVIEW

**Review by rafaqz — COMMENTED** [review]
LGTM

File: test/Manifest.toml [hunk 1]
```diff
@@ -132,7 +132,7 @@ version = "4.5.0"
 [[deps.CompilerSupportLibraries_jll]]
 deps = ["Artifacts", "Libdl"]
 uuid = "e66e0078-7015-5450-92f7-15fbd957f2ae"
-version = "1.0.2+0"
```
Comment (rafaqz) [inline test/Manifest.toml:135]: Maybe remove all these manifest changes

**Review by rafaqz — CHANGES_REQUESTED** [review]

File: test/Manifest.toml (hunk 1 above)
Comment (rafaqz) [reply test/Manifest.toml:135]: bump

File: src/Tyler.jl [hunk 2]
```diff
@@ -63,7 +63,7 @@ function Base.show(io::IO, m::MIME"image/png", map::Map)
     Makie.backend_show(map.screen, io::IO, m, map.figure.scene)
 end
 
-function Map(rect::Rect, input_cs=wgs84;
+function Map(rect::Union{Rect,Extent}, input_cs=wgs84;
```
Comment (rafaqz) [inline src/Tyler.jl:66]:
```suggestion
function Map(extent::Union{Rect,Extent}, input_cs=wgs84;
```

**Comment by rafaqz** [comment]
Damn I messed up the merge somehow.

**Review by alex-s-gardner — COMMENTED** [review]
Looks good... are we ready merge?

**Comment by rafaqz** [comment]
No, the tests are not passing!

I'm not sure how this PR changes the number of tiles downloaded

**Comment by rafaqz** [comment]
Oh you have just changed the number of tiles and the tests slightly. Fix that and we can merge

**Comment by rafaqz** [comment]
Thanks

**Comment by alex-s-gardner** [comment]
@rafaqz thank you for your patience as a battle git

**Pull request merged**